main.go

```diff
@@ -1,15 +1,64 @@
 package main
 
 import (
+<<<<<<< Updated upstream
     "log"
 
     tea "github.com/charmbracelet/bubbletea"
     . "internal/tui"
+=======
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	//. "internal/tui"
+)
+
+/*
+	This var sets up the root command and then all other commands. The root command, according to Cobra's structure, is the first thing we hit when we run the program.
+	Imagine it as an automatic constructor that's allowing us to run an instance of this program.
+*/
+var (
+	versFlag bool
+	rootCmd  = &cobra.Command{
+		Use: "put usage example here",
+		//TraverseChildren: true,
+		Short: "This program shows disk usage",
+		Long:  "Put longer version of Short here",
+		Run: func(cmd *cobra.Command, args []string) {
+			v, _ := cmd.Flags().GetBool("versFlag")
+			if v {
+				version()
+			} else {
+				fmt.Println("it brokey")
```

Comment:
If something breaks the program, we should probably use `log.Fatal` or something in the `log` lib